### ISSUE TITLE
How to retrain a searched submodel of AutoFormer?

### ISSUE THREAD

**Issue by mxjecho** [microsoft/Cream#95]
Hi, I have searched a submodel (Base) with your default training setting, but it has poor performance(78.3%). How to retrain it and what are the hyper-parameters of retraining setting?

Thanks.

**Comment by nbl97** [comment]
Hi, thanks for your interest in our work. We directly use the searched model without retraining. For your question, it seems that your model is overfitting. The large model is sensitive to the hyperparameters. You can try different warmup epochs, batch sizes, and learning rates.
